k8s/conf/nsc.yaml

```diff
@@ -8,8 +8,6 @@ spec:
       labels:
         networkservicemesh.io/app: "icmp-responder-nsc"
     spec:
-      nodeSelector:
```

Comment:
Hmm is this safe? Have you tried running this locally?

Reply:
It is safe I suppose, since we should not depend for demo container to belong to any node, and inside GKE there is no master node at all, all workers are similar. I will try to find some analogies.

Reply:
@haiodo Yeah... pegging to nodes by name is fragile... it's part of why I used to do this with Pod affinities and anti-affinities.